contracts/ShortNameClaims.sol

```diff
@@ -60,8 +84,8 @@ contract ShortNameClaims is Ownable {
      * @param claimant The address making the claim.
      * @return The claim ID.
      */
-    function computeClaimId(string memory claimed, bytes memory dnsname, address claimant) public pure returns(bytes32) {
-        return keccak256(abi.encodePacked(keccak256(bytes(claimed)), keccak256(dnsname), claimant));
+    function computeClaimId(string memory claimed, bytes memory dnsname, address claimant, string memory email) public pure returns(bytes32) {
+        return keccak256(abi.encodePacked(keccak256(bytes(claimed)), keccak256(dnsname), claimant, keccak256(bytes(email))));
```

Comment:
this may be a bit dumb question but why do we need to include email address as claimId?

Reply:
If we don't, a malicious user could submit a claim with their own email address, and the legitimate user would be unable to submit a competing claim.